docs/jsonapi-spec.md

```diff
@@ -1386,7 +1411,26 @@ value of the operation.
 
 #### $all operation
 
-TODO:
+The `$all` operation allows users to check for documents that have an array field with all of the given values.  Example:
+
+```json5
+{"locations" : { $all : ["New York", "Texas"]}}
+```
+
+If provided to a `find()`, return all the documents where the locations field contains the two values – “New York” and “Texas”. It does not matter how many more values the locations field contains. The operation will match if the specified values are present. Similarly, if even one the value is missing, the document will not be matched. So the `$all` operation is useful while retrieving data.  
+
+The `$all` is equivalent to an `$and` operation of the specified values. For example, this first statement:
+
+```json5
+{ tags: { $all: [ "ssl" , "security" ] } }
+```
+
+... is equivalent to:
+
+```json5
+{ $and: [ { tags: "ssl" }, { tags: "security" } ] }
```

Comment:
We don't support logical operator, do we need this line something we don't support?

Reply:
Removed the $and in pr 354 after merge from this main.